pkg/github/github.go

```diff
@@ -74,19 +75,21 @@ func (c *Client) SyncLabels(ctx context.Context, owner, repo string, labels []La
 	eg := errgroup.Group{}
 
 	// Delete labels.
-	for _, currentLabel := range currentLabels {
-		currentLabel := currentLabel
-		eg.Go(func() error {
-			_, ok := labelMap[currentLabel.Name]
-			if ok {
-				return nil
-			}
-			return c.deleteLabel(ctx, owner, repo, currentLabel.Name)
-		})
-	}
+	if deleteUnManaged {
```

Comment:
Use early return.

Reply:
I don't know how to use early return here we need rest of this method